src/main/java/org/launchcode/liftoffrecipetracker/Models/User.java

```diff
@@ -0,0 +1,56 @@
+package org.launchcode.liftoffrecipetracker.Models;
+
+import javax.persistence.Entity;
+import java.util.ArrayList;
+
+@Entity
+public class User extends org.launchcode.liftoffrecipetracker.models.AbstractEntity {
```

Comment:
You should be able to just extend AbstractEntity here without the whole package name. It might be thrown off because of the package issue above. When you lowercase the M on line 1 try to remove the org.launchcode.liftoffrecipetracker.models from this "extends" and see if it works.